CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ set(COMPONENT_REQUIRES
         ubirch-esp32-storage
         )
 set(COMPONENT_PRIV_REQUIRES
+        ubirch-esp32-key-storage
```

Comment:
why is this not in `COMPONENT_REQUIRES`?

Reply:
Because nothing from `ubirch-esp32-key-storage` is included in any public header of this component. I'm not sure if i'm using it in the right way, i was orienting on [ths esp-idf-docs passage](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/build-system.html#component-requirements).